docs/UserGuide.adoc

```diff
@@ -70,12 +72,14 @@ Examples:
 === Listing all persons : `list`
 
 Shows a list of all persons in the address book. +
-Format: `list`
+Format: `list` +
+Alternatives: `l` , `showall`
```

Comment:
‘viewall', 'view'

Reply:
"viewall" can be implemented, but I feel that "view" overlaps with the functionality of the select command

Reply:
oh yes